### ISSUE TITLE
Throw a window to other spaces will follow the window to that space even when the settings is unchecked.

### ISSUE THREAD

**Issue by h9nara** [ianyh/Amethyst#1306]
**Describe the bug**
Throw a window to other spaces and the focus will follow the window to that space even when the setting 'Follow thrown windows between spaces' is unchecked in preferences.

**Applications:**
Amethyst

**To Reproduce**
- Uncheck 'Follow thrown windows between spaces' in preferences
- Throw a window to another space and focus will follow

**Expected behavior**
The focus should stay at the space from which the window is thrown.

**Screenshots**

**Versions:**
 - macOS: Appears both on 12.4 and 12.5.1
 - Amethyst: 0.16.0 (94)
 
**Debug Info**
```
$ /Applications/Amethyst.app/Contents/MacOS/Amethyst --debug-info [--include-apps]
```
Note: `--include-apps` will list your manageable applications, but is optional if you don't want to list that.

**Additional context**

**Comment by mike-lloyd03** [comment]
I'm noticing this issue as well on the latest update of Amethyst and it wasn't happening before updating. I'm on MacOS 12.5.1.

I've tried checking the Follow option and then unchecking and still no effect. I've downgraded to v0.15.6 (93) for now.

**Comment by EvanDorsky** [comment]
It looks like this issue was created by PR #1209.

I resolved the issue locally by wrapping this line: https://github.com/ianyh/Amethyst/blob/cbf9612b1342ccc60a9edbaaa1dc2ec92f51ecd1/Amethyst/Managers/WindowManager.swift#L662

with the appropriate preference check:
```Swift
if UserConfiguration.shared.followWindowsThrownBetweenSpaces() {
    window.focus()
}
```

This builds and runs locally and the preference is once again respected.

@dozzman (author of PR #1209) does this change break any desirable behavior? I will submit a PR soon.

**Comment by ondvrsn** [comment]
I was hoping this fix would be included in 0.16.1 update. May I ask when this fix will be officially released?
